media-sound/teamspeak-server-bin/teamspeak-server-bin-3.0.13.8.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
1. The commit message summary line is too long.
2. The license needs to be added *before* it is used.

Reply:
Why is the commit message summary line to long? It says here https://wiki.gentoo.org/wiki/Gentoo_git_workflow 75 chars? It this wrong?
I've changed the commits now, that he license is added first.

Reply:
It's outdated. The current standard is GLEP 66 which specifies 69 for summary line. Rule of thumb is, if GitHub cuts it, it's too long.

Reply:
Thanks, that's fine for me.